dLux/dev/jwst.py

```diff
@@ -1,19 +1,7 @@
-"""
-src/dev/layers.py
------------------
-Development script for the new layers structure.
-"""
-
-from constants import *
+from dLux import Aperture, CompoundAperture
 from matplotlib import pyplot
-from typing import TypeVar
-from dLux.utils import (get_radial_positions, get_pixel_vector, 
-    get_pixel_positions)
-from abc import ABC, abstractmethod 
-import equinox as eqx
-import jax.numpy as np
-import jax 
-import functools
+from dLux.utils import (get_positions_vector, get_pixel_positions)
+from constants import *
 
 config.update("jax_enable_x64", True)
```

Comment:
We don't want to do this by default, this should be done on a script by script basis